cluster/node-pools/master-default/userdata.yaml

```diff
@@ -117,7 +117,9 @@ write_files:
           - --allow-privileged=true
           - --service-cluster-ip-range=10.5.0.0/16
           - --secure-port=443
-          - --enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,ExtendedResourceToleration,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,StorageObjectInUseProtection,PodSecurityPolicy,Priority,NodeRestriction
+          - --enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,ExtendedResourceToleration,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,StorageObjectInUseProtection,PodSecurityPolicy,Priority,NodeRestriction,EventRateLimit
+          # This file specifies the EventRateLimit admission plugin's configuration
+          - --admission-control-config-file=/etc/kubernetes/config/admission-config.yaml
```

Comment:
I'm thinking perhaps we can define a config-item `enable_eventratelimit` to be able to toggle the feature if needed. Just a thought.

Reply:
Working on adding the toggle to be able to turn the plugin off if needed.

Reply:
Added in the latest commit.